### PR TITLE
PEAA-163: added a release Github Action to automatically build and release new version

### PR DESCRIPTION
**Pull request by aleks-elkin**
- Added a GHA to automatically build and release every time we merge a PR with new version.
- Updated a README with a new flow of release process.

The command `lerna-version` will calculate a new version number based on [conventional-commits](https://www.conventionalcommits.org/en/v1.0.0/) specification and push a commit that starts from `chore(release): ` with a version number and generated CHANGELOG for all components.
After run this command developer should create a PR to merge this commit.
After merge the GHA will be triggered and will build and publish packages to NPM registry and Github packages.

I tested this GHA and flow on my personal repo: https://github.com/aleks-elkin/lerna-github-actions

### CODE REVIEW

**Review by wejendorp — APPROVED** [review]
Nice :+1:. Clean workflow. Let's try it out.

File: README.md [hunk 1]
```diff
@@ -233,13 +231,13 @@ $ npm run component-gen
...
 
-We are using [lerna to publish](https://github.com/lerna/lerna/tree/master/commands/publish#readme) our elements
+We are using [lerna to publish](https://github.com/lerna/lerna/tree/main/commands/version#readme) our elements
 
 - Create a branch for releasing。 `git branch release`
 - `git push release` and set the upstream.
-- `npm config delete @tradeshift:registry` delete the registry config so release could publish to different registries
-- `npm run release`
-- `npm config set @tradeshift:registry https://npm.pkg.github.com/` Set back npm to use github for @tradeshift packages
+- `npm run lerna-version` calculate a new version number, generate CHANGELOG.md for all components and make a commit with a predefined commit message.
```
Comment (esmailpour-hosein) [inline README.md:238]: Can we have a script to do these steps? 🤔

Comment (aleks-elkin) [reply README.md:238]: I created a simple shell script, because there are not too much to do, just create a branch and push a commit.

**Review by esmailpour-hosein — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (esmailpour-hosein) [reply README.md]: It got even better 👍 Thanks for adding it. :)

**Pull request merged**